### ISSUE TITLE
How to display scalar bar 

### ISSUE THREAD

**Issue by NahomKidane** [nmwsharp/polyscope#163]
Hi, 

Is there a way to display a scalar bar?

**Comment by nmwsharp** [comment]
Hi!

Unfortunately not; the closest we have is the colored histogram on the left pane.

This is a good feature-request though, so I will convert this issue to serve as a feature request.

**Comment by nmwsharp** [comment]
Nope, no progress, and I don't know of anyone who is working on it. Though I still think it's a good feature; I'd love to implement it personally but for now other improvements are higher priority. I'm open to a PR for this is anyone wants to give it a go!

**Comment by luocfprime** [comment]
Hi, is there a developing version of this now?
